### PR TITLE
LPS-170942 | master

### PR DESCRIPTION
**Pull request by JessieZhao0714**
https://issues.liferay.com/browse/LPS-170942

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase [hunk 1]
```diff
@@ -3392,6 +3392,138 @@ definition {
...
+		}
+
+		task ("When: User log in to able.com normally and log out") {
+			User.firstLoginUI(
+				idpName = "samlidp",
+				password = "test",
+				specificURL = "http://www.able.com:8080",
+				userEmailAddress = "usera@www.baker.com");
+
+			if (IsElementPresent.pauseIsElementPresent(locator1 = "Button#I_AGREE")) {
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase:3497]: Please define a new macro for these IF logics

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase [hunk 2]
```diff
@@ -3392,6 +3392,138 @@ definition {
...
+				SignIn.agreeToTermsOfUse();
+			}
+
+			if (IsElementPresent(locator1 = "Select#QUESTION")) {
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase:3501]: Can be combined with this IF also.

**Review by gaborlovasdotcom — CHANGES_REQUESTED** [review]
See my review comments.

**Comment by JessieZhao0714** [comment]
ci:test:sf

**Review by gaborlovasdotcom — APPROVED** [review]
LGTM

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Pull request closed without merging**